app/models/comment.rb

```diff
@@ -2,6 +2,9 @@ class Comment < ActiveRecord::Base
   belongs_to :user, class_name: 'User'
   belongs_to :post, class_name: 'Post'
 
+  validates :text, presence: true, length: { in: 1..500 }
+
+  after_save :update_comments_for_post
   def update_comments_for_post
     post.comments_counter = post.comments.count
   end
```

Comment:
[REQUIRED] I can't find a test for this method. Kindly include tests for all custom methods